src/test/operators/table_scan_between_test.cpp

```diff
@@ -0,0 +1,127 @@
+#include <tuple>
```

Comment:
Ah, I get why you do this data generation thingy, but my, I hate tests that I would have to decipher to judge whether they cover, and since deciphering is tedious I do not do it and just accept the tests as they are... :(((